### PR TITLE
Cleanup schema tests

### PR DESCRIPTION
**Pull request by MarkusH**

### CODE REVIEW

File: tests/schema/tests.py [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
 
-from django.test import TransactionTestCase
 from django.db import connection, DatabaseError, IntegrityError, OperationalError
-from django.db.models.fields import (BinaryField, BooleanField, CharField, IntegerField,
-    PositiveIntegerField, SlugField, TextField)
+from django.db.models import Model
+from django.db.models.fields import (BinaryField, BooleanField, CharField, DateTimeField,
+    IntegerField, PositiveIntegerField, SlugField, TextField)
 from django.db.models.fields.related import ForeignKey, ManyToManyField, OneToOneField
 from django.db.transaction import atomic
```
Comment (timgraham) [inline tests/schema/tests.py]: blank line after django imports

File: tests/schema/tests.py [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
-    BookWithSlug, BookWithM2M, Tag, TagIndexed, TagM2MTest, TagUniqueRename,
-    UniqueTest, Thing, TagThrough, BookWithM2MThrough, AuthorTag, AuthorWithM2MThrough,
-    AuthorWithEvenLongerName, BookWeak, Note, BookWithO2O)
+from .fields import CustomManyToManyField, InheritedManyToManyField
+from .models import (
+    Author, AuthorWithDefaultHeight, AuthorWithEvenLongerName, Book, BookWeak,
+    BookWithLongName, BookWithO2O, BookWithSlug, Note, Tag, TagIndexed,
+    TagM2MTest, TagUniqueRename, Thing, UniqueTest, new_apps
+)
+from django.test import TransactionTestCase
```
Comment (timgraham) [inline tests/schema/tests.py]: up with Django imports

File: tests/schema/tests.py [hunk 3]
```diff
@@ -26,24 +30,34 @@ class SchemaTests(TransactionTestCase):
...
+        Author, AuthorWithDefaultHeight, AuthorWithEvenLongerName, Book,
+        BookWeak, BookWithLongName, BookWithO2O, BookWithSlug, Note, Tag,
+        TagIndexed, TagM2MTest, TagUniqueRename, Thing, UniqueTest,
     ]
 
     # Utility functions
 
+    def setUp(self):
+        # local_models should contain test dependent model classes that will be
+        # automatically cleared on test tear down.
```
Comment (timgraham) [inline tests/schema/tests.py]: cleared -> removed from the app cache?

**Comment by timgraham** [comment]
Looks good.

**Comment by MarkusH** [comment]
Merged in 0204714b0bdf10d7558ee106de9a718407f3ec5a

**Pull request closed without merging**